src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -82,19 +80,20 @@ export class EntryBoxComponent extends Base implements OnInit {
   }
 
   getMyEntries() {
-    this.userEntries$ = this.usersService.getUserEntries(null, this.filterText, this.entryTypeParam);
```

Comment:
I'm glad you got rid of this; I was going to comment on it before, but didn't want to get too nit-picky when I was already commenting on an unlikely timing edge case. :) But yeah, we aren't using it, so this is good.